spyder/utils/tests/test_programs.py

```diff
@@ -36,7 +37,39 @@ def test_run_python_script_in_terminal_with_wdir_empty(tmpdir, qtbot):
     res = outfilepath.read()
     assert res == 'done'
 
-    
+@pytest.mark.skipif(os.environ.get('TEST_CI_WIDGETS', None) == True,
```

Comment:
We don't set `TEST_CI_WIDGETS` in CircleCI because we don't test our widgets there. That's why I told you to use the `CI` variable instead, which is set automatically by CircleCI, Travis and AppVeyor regardless of what we do in those services.